test/TritonGPU/wgmma-precise.mlir

```diff
@@ -0,0 +1,42 @@
+// RUN: triton-opt %s -split-input-file -convert-triton-gpu-to-llvm 2>&1  | FileCheck %s
```

Comment:
can you minimize the test and move it to:
test/Conversion/tritongpu_to_llvm_hopper.mlir
(there are already existing tests you can use to get started)

Reply:
Done.